packages/cw-croncat-core/src/types.rs

```diff
@@ -367,6 +368,17 @@ impl Task {
                         }
                     }
                 }
+                CosmosMsg::Staking(StakingMsg::Delegate {
+                    validator: _,
+                    amount,
+                }) => {
+                    // Must attach enough balance for staking
+                    if amount.amount.is_zero() {
+                        valid = false;
+                    }
+                    let add = vec![amount.clone()];
+                    amount_for_one_task.checked_add_native(&add)?;
```

Comment:
```suggestion
                    amount_for_one_task.native.find_checked_add(amount)?;
```
redundant clone here